tests/functional/cli/test_cli_variables.py

```diff
@@ -17,3 +26,59 @@ def test_list_project_variables(gitlab_cli, project):
     ret = gitlab_cli(cmd)
 
     assert ret.success
+
+
+def test_list_project_variables_with_path(gitlab_cli, project):
+    cmd = ["project-variable", "list", "--project-id", project.path_with_namespace]
+    ret = gitlab_cli(cmd)
+
+    assert ret.success
+
+
+@pytest.fixture
+def resp_get_project():
+    return {
+        "method": responses.GET,
+        "url": f"{DEFAULT_URL}/api/v4/projects/1",
+        "json": {"name": "name", "path": "test-path", "id": 1},
+        "content_type": "application/json",
+        "status": 200,
+    }
```

Comment:
I think we can have this in `tests/functional/cli/conftest.py` and remove it from here and `tests/functional/cli/test_cli.py` as it will be made available for both of them. At least if it's exactly the same which I think it is.

Reply:
Done. Good idea.